CactusTutorial.ipynb

```diff
@@ -392,11 +390,14 @@
     "\n",
     "ML_BSSN::harmonicN           = 1      # 1+log\n",
     "ML_BSSN::harmonicF           = 2.0    # 1+log\n",
-    "ML_BSSN::ShiftBCoeff         = 1\n",
-    "ML_BSSN::ShiftGammaCoeff     = 0.75\n",
-    "ML_BSSN::BetaDriver          = 2.66\n",
-    "ML_BSSN::LapseAdvectionCoeff = 0.0\n",
-    "ML_BSSN::ShiftAdvectionCoeff = 0.0\n",
+    "ML_BSSN::evolveA             = 1\n",
+    "ML_BSSN::evolveB             = 1\n",
+    "ML_BSSN::ShiftGammaCoeff     = 0.0\n",
+    "ML_BSSN::AlphaDriver         = 0.0\n",
+    "ML_BSSN::BetaDriver          = 0.0\n",  
+    "ML_BSSN::advectLapse         = 0\n",
+    "ML_BSSN::advectShift         = 0\n",
+    "ML_BSSN::MinimumLapse        = 1.0e-8\n",
```

Comment:
They do match what we had in the ET paper. On the downside those
options are really only usable for TOV oscillations as a
ShiftGammaCoeff = 0. and BetaDriver = 0 will select geodesic slicing
which tends to be a bad idea for anything that is not stationary
spacetime (certainly going to blow up for a collapsing TOV star).

Reply:
Let's add a warning that the current parameters select geodesic slicing and only work for stationary spacetimes.
I suggest to add the parameters that allow for actual collapse as comment unless it would be more complicated.

Reply:
I added a warning for now. Once we have tested the collapse, I will add the appropriate parameters as comment.